example/Makefile

```diff
@@ -41,13 +41,17 @@ run:
 	$(HELM) repo add jetstack https://charts.jetstack.io
 	$(HELM) repo update
 	$(HELM) dependency build ../charts/topolvm/
-	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml
+	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml --set image.tag=$(TOPOLVM_VERSION)
+	$(KUBECTL) apply -f ./thin-sc.yaml
 	$(KUBECTL) wait --for=condition=available --timeout=120s -n topolvm-system deployments/topolvm-controller
 	$(KUBECTL) apply -f podpvc.yaml
+	$(KUBECTL) wait --for=condition=ready --timeout=60s -n default pod -l app=example
 
 setup: $(KUBECTL)
-	$(SUDO) apt-get update
-	$(SUDO) apt-get install -y lvm2 xfsprogs
+	if ! grep fedora /etc/os-release > /dev/null; then \
```

Comment:
It is more reliable to check command directly (e.g. `which apt-get`), IMO.
BTW, we use ubuntu for development, it is a bit noisy.
In top-level Makefile, we separate tool and command setup, would you do it in the same way if you need?

Reply:
Ack, I'll probably not change anything then.